content/en/docs/tutorials/security/cluster-level-pss.md

```diff
@@ -0,0 +1,280 @@
+    # default None
+    propagation: None
+EOF
+```
+
+_Note_: Please ensure that when using Docker Desktop with KinD, the `/tmp`
+directory is added as a Shared Directory under
+`Preferences -> Resources -> File Sharing` (as seen on Docker Desktop for Mac)
+
+# Create cluster with Pod Security Standards
+
+Now we are ready to create a cluster that consumes these pod security standards
+configured with pod security admission controller:
+
+```
+$ kind create cluster --name psa-with-cluster-pss --image kindest/node:latest --config /tmp/pss/cluster-config.yaml
+Creating cluster "psa-with-cluster-pss" ...
+ ✓ Ensuring node image (kindest/node:latest) 🖼 
+ ✓ Preparing nodes 📦  
+ ✓ Writing configuration 📜 
+ ✓ Starting control-plane 🕹️ 
+ ✓ Installing CNI 🔌 
+ ✓ Installing StorageClass 💾 
+Set kubectl context to "kind-psa-with-cluster-pss"
+You can now use your cluster with:
+
+kubectl cluster-info --context kind-psa-with-cluster-pss
+
+Have a question, bug, or feature request? Let us know! https://kind.sigs.k8s.io/#community 🙂
+
+
+$ kubectl cluster-info --context kind-psa-with-cluster-pss
+Kubernetes control plane is running at https://127.0.0.1:63855
+CoreDNS is running at https://127.0.0.1:63855/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+
+To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+```
+
+# Create pod
+
+Creating a pod with minimal configuration in default namespace should now be
+successful with warnings for `restricted` pod security standards:
+
+```
+$ cat <<EOF > /tmp/pss/nginx-pod.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+spec:
+  containers:
+    - image: nginx
+      name: nginx
+      ports:
+        - containerPort: 80
+EOF
+
+
+$ kubectl apply -f /tmp/pss/nginx-pod.yaml
+Warning: would violate PodSecurity "restricted:latest": allowPrivilegeEscalation != false (container "nginx" must set securityContext.allowPrivilegeEscalation=false), unrestricted capabilities (container "nginx" must set securityContext.capabilities.drop=["ALL"]), runAsNonRoot != true (pod or container "nginx" must set securityContext.runAsNonRoot=true), seccompProfile (pod or container "nginx" must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost")
+pod/nginx created
+```
+# Clean up
+
+Run `kind delete cluster -name psa-with-cluster-pss` and 
+`kind delete cluster -name psa-wo-cluster-pss` to delete the clusters we 
+created.
+
+# Bringing it all together
+
+Running the commands in this
+[gist](https://gist.github.com/PushkarJ/9f7a0045f4bec31097bdd1e9db0f2f6e)
+that were discussed in this tutorial will do all these steps in one go:
+
+1. Create a Pod Security Standards based cluster level Configuration
+2. Create a file to let API server consumes this configuration
+3. Create a cluster that creates an API server with this configuration
+4. Set kubectl context to this new cluster
+5. Create a minimal pod yaml file
+6. Apply this file to create a Pod in the new cluster
+
```

Comment:
Add a ## {{% heading "whatsnext" %}} 

and a link to:

Pod Security Admission (/docs/concepts/security/pod-security-admission/)
Pod Security Standards (/docs/concepts/security/pod-security-standards/)
Applying Pod Security Standards at the namespace level (/docs/tutorials/security/ns-level-pss/)

Reply:
done